src/js_set.hpp

```diff
@@ -162,6 +162,7 @@ struct SetClass : ClassDefinition<T, realm::js::Set<T>, CollectionClass<T>> {
 
      MethodMap<T> const methods = {
         {"add", wrap<add>},
+        {"get", wrap<get>},
```

Comment:
Is this required?  We had a long discussion about whether to expose a `get` function during the main implementation work.

Reply:
It is used in the test, I enabled?

Reply:
Yes, but the test was written and commented out prior to the decision to remove the `get` accessor was taken.
However, since `Set` is also a `Collection`, there are other ways to do index-based accession without exposing `get` on `Set`.

Reply:
I will change it to `[]`.